project_multifamily_beta/housing_characteristics/Window Areas.tsv

```diff
@@ -1,2 +1,10 @@
-Option=F15 B15 L15 R15
-1.000000
+Dependency=Geometry Building Type ACS	Option=F6 B6 L6 R6	Option=F9 B9 L9 R9	Option=F12 B12 L12 R12	Option=F15 B15 L15 R15	Option=F18 B18 L18 R18	Option=F30 B30 L30 R30
+Mobile Home	0.309	0.210	0.354	0.027	0.096	0.004
+Single-Family Attached	0.309	0.210	0.354	0.027	0.096	0.004
+Single-Family Detached	0.309	0.210	0.354	0.027	0.096	0.004
+2 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+3 or 4 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+5 to 9 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+10 to 19 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+20 to 49 Unit	0.122	0.063	0.432	0.027	0.249	0.107
+50 or more Unit	0.0	0.0	0.0	0.0	0.0	1.0
```

Comment:
We actually already have a breakout of low vs. high for these building types. See https://github.com/NREL/OpenStudio-BuildStock/blob/master/project_multifamily_beta/housing_characteristics/Geometry%20Building%20Number%20Units%20HL.tsv. Please change your dependency to this instead.

Reply:
Fixed with https://github.com/NREL/OpenStudio-BuildStock/pull/393/commits/51ea1d694c4141739025eb355da7392486b47331